docs/user-guide/sdks-using.md

```diff
@@ -4,9 +4,11 @@ Leverage the Zowe Client Software Development Kits (SDKs) to build client applic
 
 The SDKs include programmatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
 
-The following SDKs are available.
+The following SDKs are available: 
+  
 - Zowe Node.js Client SDK
 - Zowe Python Client SDK
+- Zowe Java Client SDK
```

Comment:
Should we have the Java SDK before the python one (given theirs respective statuses)? 😋

Reply:
I moved it up the list in another recent checkin.